app/src/main/java/de/fraunhofer/fokus/OpenMobileNetworkToolkit/LoggingService.java

```diff
@@ -503,6 +541,16 @@ private void stopRemoteInfluxDB() {
             }
         }
 
+        if (remoteInfluxHandlerThread != null) {
+            remoteInfluxHandlerThread.quitSafely();
+            try {
+                remoteInfluxHandlerThread.join();
+            } catch (InterruptedException e) {
+                Log.e(TAG, "Exception happened!! "+e, e);
+            }
+            remoteInfluxHandlerThread = null;
+        }
```

Comment:
this would have been an ideal kotlin extension function :D

Reply:
nothing to be done about this, was just saying since the same thing is used over and over... resolving.